dist/js/nya-bs-select.js

```diff
@@ -553,7 +553,7 @@ nyaBsSelect.directive('nyaBsSelect', ['$parse', '$document', '$timeout', '$compi
         dropdownMenu.append(noSearchResult);
       }
 
-      if (tAttrs.actionsBox === 'true' && isMultiple) {
+      if (typeof tAttrs.actionsBox !== 'undefined' && isMultiple) {
```

Comment:
Because the decision to add the box to the DOM is made at compile time (pre-binding) it eagerly creates the element in case the attribute evaluates to true at link time where the decision to show or hide is actually made (see line 793 & 849). It's an optimisation to not create the elements unless the relevant attribute exists. You should be able to remove both those clauses you highlighted and it should all carry on working as expected, the elements will just always be created.